src/ui/osx/TogglDesktop/Feature/TimeEntryEditor/DesktopLibraryBridge.h

```diff
@@ -40,8 +40,6 @@ NS_ASSUME_NONNULL_BEGIN
 							 projectID:(uint64_t)projectID
 						   projectGUID:(NSString *)projectGUID;
 
-- (void)togglEditor;
```

Comment:
I remove this method since it's never used in any place. Meanwhile, the name is mis-used.

Reply:
makes sense 👍